examples-esp32c3/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2021"
 embedded-svc.workspace = true
 bleps.workspace = true
 bleps-macros.workspace = true
-embassy-executor .workspace = true
+embassy-executor = { workspace = true, features = ["arch-riscv32"] }
```

Comment:
I don't think we need this feature at the dependency?!
only on the esp32c3 feature like the rest of the examples